src/xunit.performance.execution/xunit.performance.execution.csproj

```diff
@@ -7,7 +7,7 @@
   <PropertyGroup>
     <AssemblyTitle>xunit.performance.execution</AssemblyTitle>
     <Description>xUnit Performance Execution</Description>
-    <TargetFramework>netstandard1.3</TargetFramework>
+    <TargetFramework>netstandard1.5</TargetFramework>
```

Comment:
Does this mean we're dropping support for 1.3? Or has this been the case from the start?